blog/2024-07-23-july-releases.mdx

```diff
@@ -0,0 +1,233 @@
+- Add an `envvars_force_uppercase` setting which defaults to `True`, uppercasing all environment variables (thereby justifying `conda`'s current behaviour); when `envvars_force_uppercase` is set to `False`, conda will only save preserved-case variable names.
+- Alias `conda env list` command to `conda info --envs`.
+- Replace calls to `logger.warn` with `logger.warning`.
+
+### 🔧 What Got Fixed? 🔧
+
+- Improve treatment of logger levels.
+- Mask authentication details in `conda-meta/*.json` metadata.
+- Mask Anaconda.org tokens in verbose logs.
+- Fix parsing error when history file only contains a single commented line.
+- Add missing `emscripten` and `wasi` entries to the recognized platforms, and `wasm32` to the recognized architectures.
+- Fix checksum comparisons in `conda.gateways.connection.download.download()` to be case insensitive.
+- Fix caching when `repodata.json` contains `\r\n` line endings.
+- Fix `conda.core.portability.binary_replace` not matching chunks that end with `\n`.
+
+### 🌅 What's Marked for [Deprecation](https://github.com/conda-incubator/ceps/blob/main/cep-9.md)? 🌅
+
+- Mark `conda.gateways.logging.initialize_root_logger` as pending deprecation.
+- Mark `conda.cli.main_env_list.execute` as pending deprecation. Use `conda.cli.main_info.execute` instead.
+- Revert `--all` deprecation in `conda info`.
+- Mark `conda.exports.iteritems` as pending deprecation. Use builtin `dict.items()` instead.
+- Mark `conda.exports.Completer` as pending deprecation.
+- Mark `conda.exports.InstalledPackages` as pending deprecation.
+- Mark `conda.exports.KEYS` as pending deprecation.
+- Mark `conda.exports.KEYS_DIR` as pending deprecation.
+- Mark `conda.exports.hash_file` as pending deprecation.
+- Mark `conda.exports.verify` as pending deprecation.
+- Mark `conda.exports.symlink_conda` as pending deprecation. Use `conda.activate` instead.
+- Mark `conda.exports._symlink_conda_hlp` as pending deprecation. Use `conda.activate` instead.
+- Mark `conda.exports.win_conda_bat_redirect` as pending deprecation. Use `conda.activate` instead.
+- Mark `conda.utils.win_path_to_cygwin` as pending deprecation. Use `conda.common.path.win_path_to_unix` instead.
+- Mark `conda.utils.cygwin_path_to_win` as pending deprecation. Use `conda.utils.unix_path_to_win` instead.
+- Mark `conda.utils.translate_stream` as pending deprecation.
+- Mark `conda.utils.unix_shell_base` as pending deprecation. Use `conda.activate` instead.
+- Mark `conda.utils.msys2_shell_base` as pending deprecation. Use `conda.activate` instead.
+- Mark `conda.utils.shells` as pending deprecation. Use `conda.activate` instead.
+
+### 📄 What's New in Documentation? 📄
+
+- Clarify proxy server configuration in documentation.
+- Add type hints and doc strings to `conda.core.envs_manager`.
+- Add logging overview as deep-dive.
+- Update conda cheatsheet text and add it directly to cheatsheet page.
+- Add cheatsheet PDF download to cheatsheet page.
```

Comment:
Should probably add "updated" to this (which I didn't think about when I made the news item). We've had PDF downloads for a while.
```suggestion
- Add updated cheatsheet PDF download to cheatsheet page.
```